docs/menu.cue

```diff
@@ -384,6 +384,11 @@
 									text: "Use Firebase Authentication"
 									path: "/how-to/firebase-auth"
 									file: "how-to/firebase-auth"
+								}, {
```

Comment:
Should we make this a sub-menu of the Kubernetes docs instead?

Reply:
Another level under https://encore.dev/docs/deploy/kubernetes?

Reply:
Yeah exactly